cli/ballerina-cli/src/test/resources/test-resources/command-outputs/unix/build-tool-non-existent-subcommand.txt

```diff
@@ -0,0 +1,5 @@
+
+Executing Build Tools
+ERROR [dummy_tool.nonexistent] Build tool command 'dummy_tool.nonexistent' not found
+WARNING: Execution of the build tool 'dummy_tool.nonexistent' for 'main_dummy' is skipped due to errors
```

Comment:
```suggestion
WARNING: Execution of 'dummy_tool.nonexistent:main_dummy' is skipped due to errors
```
Since we print `Executing build tools` message at the beginning, let's remove the repeating parts.

Reply:
Fixed with 9c900ee382a0ef596c5cbe9e2298352bc5cbfa89